src/Microsoft.DotNet.Interactive/KernelExtensions.cs

```diff
@@ -110,7 +111,8 @@ public static TKernel UseImportMagicCommand<TKernel>(this TKernel kernel)
 
     public static async Task LoadAndRunInteractiveDocument(
         this Kernel kernel,
-        FileInfo file)
+        FileInfo file,
+        KernelCommand parentCommand)
```

Comment:
This should probably be optional. There won't always be a parent command for this method, e.g. in automation scenarios.